cypress/integration/misc/discount_schema_to_bpartner_spec.js

```diff
@@ -0,0 +1,26 @@
+import { DiscountSchema } from '../../support/utils/discountschema';
```

Comment:
this test is about about a customer discount schema, right? 
In that cause, i think it should be reflected in the test file name..or maybe in the folder the test is placed in?